ql/src/test/queries/clientpositive/acid_direct_delete.q

```diff
@@ -0,0 +1,18 @@
+set hive.mapred.mode=nonstrict;
```

Comment:
why add a new q.test instead of extending the existing `acid_direct_update_delete.q`?

Reply:
As this is a very specific use case, I wrote it in different q test.

Reply:
it doesn't need a separate test, it's an overkill

Reply:
Agree on that. I will rewrite it.